webapp/website/ts/components/services/services/ServicesFct.ts

```diff
@@ -64,6 +64,17 @@ export class ServicesFct {
 		}
 	}
 
+	getWindowsStatus(): angular.IPromise<boolean> {
+		return this.servicesSvc.getAll().then((services: IService[]) => {
+			let state = false;
+			services.forEach(function(service: IService) {
+				if (service.Ico === "windows" && service.Status === "running") {
```

Comment:
This loop implies all windows must be running is order for state to be on. Should not only one be enough ?